src/components/forms/Range/Range.stories.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Meta, Story } from '@storybook/react/types-6-0';
+
+import Range from './Range';
+import { RangeProps } from './Range.types';
+
+export default {
+  title: 'components/forms/Range',
+  component: Range,
+  argTypes: {
+    min: { control: 'number' },
+    max: { control: 'number' },
+    defaultValue: { control: 'number' },
+    isProgressRight: { control: 'boolean' },
+    isDisabled: { control: 'boolean' },
+  },
+} as Meta;
+
+export const playground: Story<
```

Comment:
Please add `const RangeTemplate` and use the notation we started using elsewhere https://www.figma.com/file/5FMXj5ydt1i7N6gU01EGVq/DS-Snowflakes?node-id=7030%3A17505
```jsx
const RangeTemplate: Story<
  RangeProps & React.InputHTMLAttributes<HTMLInputElement>
> = (args) => <Range {...args} aria-label="Input" />;

export const playground = RangeTemplate.bind({});
playground.parameters = {
  chromatic: { disable: true },
};
playground.args = { min: 0, max: 100 };
...